src/routes/resources/+page.svelte

```diff
@@ -194,12 +234,25 @@
   </div>
 </details>
 
-<div
-  class="grid xl:grid-cols-3 md:grid-cols-2 sm:grid-cols-1 gap-x-4 gap-y-4 mt-3"
+<button
+  class="p-2 hover:bg-green-500 hover:text-white cursor-pointer fixed ease-in-out bottom-10 right-10 z-50 rounded-lg border-2 border-green-500 text-green-500"
+  on:click={scrollToTop}
+  class:hidden={!showButton}
 >
-  {#each displayedResources as resource}
-    <ListItem {...resource} />
-  {:else}
-    <div>No resources here!</div>
-  {/each}
-</div>
+  Back to Top
```

Comment:
Love this (also how it only shows up when scrolling somewhat down the page).

Could we replace this with an icon? https://icons.getbootstrap.com/icons/chevron-up/ Round button, with green background and white path would be good (you can copy paste the purple link button component and adjust; or leave it to me and I'll patch)

Reply:
sure